clerk/organizations.go

```diff
@@ -166,6 +166,13 @@ type ListOrganizationMembershipsParams struct {
 	OrganizationID string
 	Limit          *int
 	Offset         *int
+	UserIDs        []string
+	EmailAddresses []string
+	PhoneNumbers   []string
+	Usernames      []string
+	Web3Wallets    []string
+	OrderBy        string
+	Query          string
```

Comment:
🔧 I believe these (`OrderBy` & `Query`) are optional so the type should be `*string` instead